src/wiseagents/graphdb/lang_chain_wise_agent_graph_db.py

```diff
@@ -97,6 +97,7 @@ class Neo4jLangChainWiseAgentGraphDB(LangChainWiseAgentGraphDB):
     and a corresponding Neo4j vector database.
     """
     yaml_tag = u'!wiseagents.graphdb.Neo4jLangChainWiseAgentGraphDB'
+    expects_underscores_in_yaml_keys = False
```

Comment:
This doesn't need to block this PR from being merged but just a question. Is there a reason why this is only added in this file? I don't see it being used anywhere. If it's not needed, we can remove it in a separate PR after merging.

Reply:
Ah yeah, I was thinking I had to phase in the change but it worked out in another way https://github.com/wise-agents/wise-agents/pull/364